iocore/aio/AIO.cc

```diff
@@ -423,17 +423,15 @@ ink_aio_thread_num_set(int thread_num)
 void *
 AIOThreadInfo::aio_thread_main(AIOThreadInfo *thr_info)
```

Comment:
Not directly relevant to this PR, but `thr_info` is redundant, it's  the same as the `this` pointer in this function.